src/scripts/commands/database.ts

```diff
@@ -0,0 +1,205 @@
+// Commands related to dhis2 database direct manipulation.
+// The generated sql files can be run for example with: d2-docker run-sql <file>
+
+import _ from "lodash";
+import fs from "fs";
+import { command, subcommands, positional, flag, option, optional, string } from "cmd-ts";
+import { HierarchyLevel, OrgUnitPath } from "scripts/common";
+
+export function getCommand() {
+    const removeOrgUnitLevelCommand = command({
+        name: "remove-orgunit-level",
+        description: "Create a sql file that removes organisation units below a hierarchy level",
+        args: {
+            level: positional({
+                type: HierarchyLevel,
+                description: "Maximum hierarchy level to keep",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
+            const sqlCommands = removeOrgs(`hierarchylevel > ${args.level}`);
+            safeWrite(outputFile, sqlCommands, args.overwrite);
+            process.exit(0);
+        },
+    });
+
+    const removeOrgUnitChildrenCommand = command({
+        name: "remove-orgunit-children",
+        description: "Create a sql file that removes all children of an organisation unit",
+        args: {
+            path: positional({
```

Comment:
Same, here it's even less clear what the positional argument means, so better to have it named.

Reply:
I changed it so you can add either option, and it will fail if both are missing. I suspect there may be a more idiomatic way than the way I did it, though.

Reply:
It looks good to me